apps/sms/test/unit/drafts_test.js

```diff
@@ -464,57 +403,54 @@ suite('Drafts', function() {
     });
 
     test('Store fresh drafts', function() {
-      Drafts.add(d1);
-      Drafts.add(d2);
-      Drafts.add(d5);
+      Drafts.add(threadDraft1);
+      Drafts.add(threadDraft2);
+      Drafts.add(draft1);
 
       sinon.assert.calledThrice(Drafts.store);
     });
 
     test('Store draft with distinct content', function() {
-      Drafts.add(d5);
-      // d6 is almost the same as d5, b/w different content
-      Drafts.add(d6);
+      Drafts.add(draft1);
+      // threadDraft5 is almost the same as draft1, b/w different content
+      Drafts.add(threadDraft5);
 
       sinon.assert.calledTwice(Drafts.store);
     });
 
     test('Store draft with distinct subject', function() {
-      Drafts.add(d5);
-      // d7 is almost the same as d5, b/w different subject
-      Drafts.add(d7);
+      Drafts.add(draft1);
+      // draft2 is almost the same as draft1, b/w different subject
+      Drafts.add(draft2);
 
       sinon.assert.calledTwice(Drafts.store);
     });
 
     test('Load drafts, has stored data', function(done) {
       this.sinon.stub(asyncStorage, 'getItem').yields([
-        [42, [d1]],
-        [44, [d2]],
-        [null, [d5, d6, d7]]
+        [42, [threadDraft1]],
+        [44, [threadDraft2]],
+        [null, [draft1, draft2]]
```

Comment:
We have only two threadless drafts, third one (with thread id) here didn't look right.